server/controllers/accounts.controller.js

```diff
@@ -9,52 +12,76 @@ const AccountsController = {
    *
    * @param  {object} res - response
    *
-   * @return {json} - jsonObject containing status code and or error
+   * @return {json} - jsonObject containing status code and data or error
    *
    * Route: POST: /accounts
    *
    * */
   async create(req, res) {
     const { error, isValid } = validateCreateBankAccountInput(req.body);
     if (!isValid) {
-      return res.status(400).json({
-        status: 400,
-        error,
-      });
+      return res.status(400).json({ status: 400, error });
     }
 
-    const newAccount = await Account.create(req.body);
-    return res.status(201).json({
-      status: 201,
-      data: {
-        accountNumber: newAccount.accountNumber,
-        firstName: newAccount.firstName,
-        lastName: newAccount.lastName,
-        email: newAccount.email,
-        type: newAccount.type,
-        openingBalance: newAccount.balance,
-      },
-    });
+    const {
+      firstName, lastName, email, type,
+    } = req.body;
+
+    // Generate account account number
+    const date = new Date();
+    const accountNumber = date.getTime();
+
+    // Check if user has a registered account
+    User.findOne('email', email)
+      .then((result) => {
+        if (!result || result.rows.length < 1) {
+          return res.status(400).json({
+            status: 400,
```

Comment:
Status code here is not correct. Please change it.

Reply:
Changed to
#### Not Found.

Reply:
I do not understand what you mean by Not found. I cannot see it in the code.